src/_posts/security/procedures/2000-01-01-external-testing.md

```diff
@@ -11,10 +11,17 @@ tags: security pentest load-test procedures
 
 You can run a load test / smoke test / endurance test on your application under the following conditions:
 
-- Ask the support using support [at] scalingo.com or use the support chat
-- You should communicate the metrics of your test to the support (number of connections...)
-- The test should only be run during business hours - 09:00 to 12:00 and 14:00 to 18:00 (CET/CEST, Europe/Paris timezone)
-- The test should have specific start and end hours
+1. Ask the support using [support@scalingo.com](mailto:support@scalingo.com) or use the support chat
+2. Your request should include the following information:
+  - The application name on Scalingo,
+  - The Scalingo region where the application is hosted,
+  - The source IPs of the requests,
+  - The target public URLs that will be used,
+  - Number of requests per minute,
+  - Size of the payload of each request,
+  - Time slot the load test is planned. Note that the test should only be run during business hours - 09:00 to 12:00 and 14:00 to 18:00 (CET/CEST, Europe/Paris timezone). The test must have a specific start and end hour.
```

Comment:
"Time slot **in which** the test is planned"
"... specific start and end **time**"